src/api/server/authn.rst

```diff
@@ -345,3 +345,124 @@ headers to CouchDB with related requests:
 
 Note that you don't need to request :ref:`session <api/auth/session>`
 to be authenticated by this method if all required HTTP headers are provided.
+
+.. _api/auth/jwt:
+
+JWT Authentication
+====================
+
+.. note::
+    To use this authentication method make sure that the
+    ``{chttpd_auth, jwt_authentication_handler}`` value is added to the
+    list of the active :config:option:`chttpd/authentication_handlers`:
+
+    .. code-block:: ini
+
+        [chttpd]
+        authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, jwt_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+``JWT authentication`` enables CouchDB to use externally generated JWT tokens
```

Comment:
[grammar nit]

```suggestion
``JWT authentication`` enables CouchDB to use externally-generated JWT tokens
```

Reply:
again, this is not conventionally hyphenated, I don't understand the suggestion that it should be.

Reply:
It is correct to hyphenate here because it is an adjective phrase.  You would not hyphenate when it's used as an adverbial phrase. i.e. "The JWT is externally generated".